src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -171,17 +175,21 @@
 
     private static final FactoryProvider factoryProvider;
     private static final LoadingCache<Class<?>, EventFactory<?>> factories;
+    private static final Deque<EventFactoryPlugin> plugins;
 
     static {
         factoryProvider = new ClassGeneratorProvider("org.spongepowered.api.event.impl");
```

Comment:
How would the target package be discovered, since right now it's a different package from `SpongeEventFactory`.

This could probably go in a separate PR.